modules/xmpp/AVModeration.ts

```diff
@@ -1,8 +1,9 @@
 import { getLogger } from '@jitsi/logger';
 import { $msg } from 'strophe.js';
-
+import ChatRoom from "./ChatRoom";
```

Comment:
Doh I'm not sure why the linger didn't catch this.

Can you please make it single quotes?

Also put it right before the XMPP import please.

I'll look into what's up with the linter...